include/improvserial.h

```diff
@@ -154,19 +147,23 @@ class ImprovSerial
         __attribute__((format(printf, 2, 3)))
         void log_write(const char* format, ...)
         {
+            constexpr int bufferSize = 256;
+            char lineBuffer[bufferSize];
+
             auto file = SPIFFS.open(IMPROV_LOG_FILE, FILE_APPEND);
             va_list args;
 
             va_start(args, format);
 
             // We shifted the printf parameter check to ourselves, so we can suppress the warning here
             #pragma GCC diagnostic ignored "-Wformat-nonliteral"
-            file.printf(format, args);
+            vsnprintf(lineBuffer, bufferSize, format, args);
```

Comment:
This change (and those leading and trailing it) actually make the arguments after the format string get through to the log lines that are written.

Reply:
No, they are not. Good catch, thanks! I will include their removal with a change I'll add to make the hostname a device setting. After testing with 3 different devices on different APs I now have 4 IP addresses assigned to different "Something"s, and it just doesn't make sense for a code-level change to be the only way to configure it.